ui/app/css/itcss/components/popover.scss

```diff
@@ -0,0 +1,328 @@
+
+
+.popover-content {
+  // not sure how this is gonna work in other browsers?
+  ::-webkit-scrollbar {
+    width: 6px;
+  }
+
+  ::-webkit-scrollbar-thumb {
+    -webkit-border-radius: 10px;
+    border-radius: 10px;
+    background: #C4C4C4;
+  }
+
+  position: relative;
+  width: 328px;
+  height: 564px;
+
+  box-shadow: 0px 4px 30px rgba(0, 0, 0, 0.25);
+  border-radius: 10px;
+  background: white;
+  z-index: 2;
+  left: calc(50% - 328px/2);
+  top: 10px;
+
+  .header {
+    h2 {
+      font-family: Roboto;
```

Comment:
Nite: See above comment on font mixins. The font mixins are added in `variables.scss`. An example usage is https://github.com/MetaMask/metamask-extension/blob/develop/ui/app/components/app/permission-page-container/index.scss (`@extend...`)